.github/workflows/danger.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Danger
-        uses: danger/swift@3.0.0
+        uses: f-meloni/danger-swift-swiftlint-action@0.2.8
```

Comment:
If you use `docker://frmeloni/danger-swift-with-swiftlint:1.1.0` you should save more than 6 minutes of build